src/tools/getInputPropsMiddleware.tsx

```diff
@@ -0,0 +1,6 @@
+import { Trans } from 'next-i18next';
+
+export const getInputPropsMiddleware = (inputProps: { error: string } & any) => ({
```

Comment:
Is this ``& any`` really needed here?

Reply:
Yeah this any is used because the getInputProps method from mantine has the same type.